modules/ROOT/pages/database-administration/aliases/remote-database-alias-configuration.adoc

```diff
@@ -193,6 +200,123 @@ If there is a failure when reading the keystore file, investigate the `debug.log
 In case it is not possible to connect to the remote alias after its creation, verify its settings by connecting to the remote database at https://browser.neo4j.io/ or at your local browser.
 ====
 
+[role=label--new-2025.10]
+[[setup-example-credential-forwarding]]
+== Setup example with OIDC credential forwarding
+
+In order to use OIDC credential forwarding, both *DBMS A* and *DBMS B* need to support the same OIDC identity provider, see the xref:authentication-authorization/sso-integration.adoc[SSO integration] on how to enable OIDC.
+
+In this example, _Alice_ is an administrator and _Carol_ is a user who needs access to a database managed by _Bob_:
+
+
+image::remote-alias-credential-forwarding-overview.svg[title="Overview of the required remote database alias setup when using oidc credential forwarding", role="middle"]
+
+In the example above, _Carol_ logs in to *DBMS A* through an OIDC compliant identity provider by offering a token from the provider.
+The token is used to set the username and determine the identity provider groups of the user.
+_Alice_ is the admin of *DBMS A* and has set up SSO for the identity provider and configured the mapping of the identity provider groups to the Neo4j roles, such that _Carol_ can use the remote database alias to connect to the remote database `Db1`.
+Additionally, _Bob_ needs to configure the *DBMS B* to support SSO with the same identity provider used by _Carol_ to log in to *DBMS A*. _Bob_ also needs to configure the mapping of the identity provider groups to the Neo4j roles such that the _Carol's_ identity provider groups gives the appropriate privileges to access `Db1` on the *DBMS B*.
+
+
+[CAUTION]
+====
+While it is permitted to use different OIDC configurations across distinct DBMS instances (e.g., local vs. target), users must be aware of the resulting privilege disparity.
+A user's effective permissions are not dictated by the identity provider groups alone, but by the mapping of those groups to the roles defined within each specific Neo4j DBMS, see xref:authentication-authorization/sso-integration.adoc#auth-sso-map-idp-roles[Map the identity provider groups to the Neo4j roles].
+Crucially, if the OIDC configuration settings differ between the local DBMS and the target DBMS, the user will have different effective privileges on those systems.
+This configuration independence can lead to privilege inconsistency (e.g., over-privileging or unexpected access denial).
+====
+
+=== Configure a remote DBMS (_Bob_)
+
+
+In the suggested example, there are two administrators: _Alice_ and _Bob_.
+_Bob_ is the administrator responsible for the setup of the remote DBMS, which includes the following steps:
+
+. Set up SSO and support for the identity provider.
+. Map the identity provider groups to the Neo4j roles. If you don’t want specific users to access `Db2`, here is where you set it.
```

Comment:
Maybe this is a good point to link to
https://neo4j.com/docs/operations-manual/current/tutorial/tutorial-sso-configuration/

Same for "Configure local DBMS" below